packages/sui-ssr/server/hooksFactory/index.js

```diff
@@ -75,6 +75,9 @@ export const hooksFactory = async () => {
     [TYPES.ROUTE_MATCHING]: async (req, res, next) => {
       const startRouteMatchingTime = process.hrtime()
       const {performance = {}, url} = req
+      const site = siteByHost(req)
+      const siteRoutes =
+        isMultiSite && isFunction(routes) ? routes({site}) : routes
```

Comment:
I think we could check `site` var instead of `isMultiSite` because `site` will only exist if multisite feature is enabled: https://github.com/SUI-Components/sui/blob/master/packages/sui-ssr/server/utils/factory.js#L47